stardog/files/jmx-exporter.yaml

```diff
@@ -9,6 +9,10 @@ rules:
     name: stardog_database_$2
     labels:
       database: $1
+  - pattern: Stardog<00=databases, 01=(.+?), name=(.+)><>(Count|Value)
+    name: stardog_database_$2
```

Comment:
Is it possible to reuse the name?

Reply:
Yes, it shouldn't collide. There's an open ticket with Stardog to unify the metric names so we can get rid of this.